.pipelines/nuget.yml

```diff
@@ -19,6 +19,15 @@ parameters:
   type: boolean
   default: false
 
+- name: NugetVersionSuffix
+  displayName: Update nuget version suffix (only needed if publishing to nuget.org)
+  type: string
+  default: alpha
+  values:
+    - alpha
+    - beta
+    - rc
```

Comment:
can users add a new value here via ADO paramenter?

Reply:
not at the minute, i can add that. was originally thinking that might allow for unwanted/accidental inconsistent naming is that fine?